examples/calculations/QVector.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2015-2022 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""
+========
+Q-Vector
+========
+
+Use `metpy.calc.q_vector`.
+
+This example demonstrates the q_vector calculation by computing them from the example xarray
+Dataset and plotting using Matplotlib.
+"""
+import matplotlib.pyplot as plt
+
+import metpy.calc as mpcalc
+from metpy.cbook import example_data
+from metpy.units import units
+
+# load example data
+ds = example_data()
+
+# Calculate the temperature advection of the flow
+tadv = mpcalc.advection(ds.temperature, ds.uwind, ds.vwind)
+
+# Calculate the q-vectors
+u_qvect, v_qvect = mpcalc.q_vector(ds.uwind, ds.vwind, ds.temperature, 850 * units.hPa)
+print(u_qvect.data.units)
```

Comment:
Should we break the cell here so that the print out isn't after the plot? Or is this leftover debugging?

Reply:
Left over from debugging to get the right units for the title!